procedures: starting a new workspace (#2227)
codenvy/che-docs: modules/user-guide/pages/url-parameter-for-the-workspace-ide.adoc

:_content-type: CONCEPT
:description: URL parameter for the workspace IDE
:keywords: url-parameter-workspace-ide, workspace-ide, url-workspace-ide, url-parameter-workspace-editor, workspace-editor, url-workspace-editor
:navtitle: URL parameter for the workspace IDE
// :page-aliases:

[id="url-parameter-for-the-workspace-ide_{context}"]
= URL parameter for the workspace IDE

> Che-Theia is currently the default integrated development environment (IDE) that a workspace loads with if no IDE is specified as a URL parameter as part of the URL for starting a new workspace.

Reviewer: ```suggestion
Che-Theia is currently the default integrated development environment (IDE) loaded by a workspace if no IDE is specified as a URL parameter as part of the URL for starting a new workspace.
```
